backend/routes/profile.py

```
@@ -0,0 +1,93 @@
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException, UploadFile, File, Form
+from sqlalchemy.orm import Session
+import models.user as models
+from database.session import SessionLocal
+from models.user import User  # ✅ Correct model import
```

Ruff (0.8.2)

`models.user.User` imported but unused

Remove unused import: `models.user.User`

(F401)